### PR TITLE
[ICDS Dashboard] Added view, sql and test case for kpi api

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/1112385193248823/1167712643236679/f

##### SUMMARY
Added view, sql and test case for kpi api

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/migrations/sql_templates/database_views/kpi_api.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
+DROP VIEW IF EXISTS kpi_api_view CASCADE;
+CREATE VIEW kpi_api_view AS
```
Comment (Rohit25negi) [inline custom/icds_reports/migrations/sql_templates/database_views/kpi_api.sql]: well. kpi(key performance indicators) is pretty general name. Would be great to make it specific like pmo_api_view. something like that.

File: custom/icds_reports/models/views.py [hunk 2]
```diff
@@ -1392,3 +1392,43 @@ class Meta(object):
...
+    Contains rows for KPI API.
+    """
+    district_id = models.TextField(blank=True, null=True)
+    district_name = models.TextField(blank=True, null=True)
+    district_site_code = models.TextField(blank=True, null=True)
+    state_id = models.TextField(blank=True, null=True)
+    state_name = models.TextField(blank=True, null=True)
+    state_site_code = models.TextField(blank=True, null=True)
+    aggregation_level = models.IntegerField(blank=True, null=True)
+    district_map_location_name = models.TextField(blank=True, null=True)
```
Comment (Rohit25negi) [inline custom/icds_reports/models/views.py]: i think you can remove these map_location_names

**Review by Rohit25negi — APPROVED** [review]
Looks good. Left two comments. Not blocking but good to have them before we merge this.

File: custom/icds_reports/models/views.py (hunk 2 above)
Comment (challabeehyv) [reply custom/icds_reports/models/views.py]: removed

File: custom/icds_reports/migrations/sql_templates/database_views/kpi_api.sql (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/migrations/sql_templates/database_views/kpi_api.sql]: done

**Comment by Rohit25negi** [comment]
Thanks for the followup.

**Pull request merged**